tests/libvmi/list.go

```diff
@@ -30,7 +30,8 @@ import (
 	"kubevirt.io/kubevirt/tests/libpod"
 )
 
-func GetRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, namespace string) (string, error) {
+func GetRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient,
```

Comment:
can you squash this commit into the commit that introduced these linting errors ?